security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -335,6 +356,52 @@ func (sc *SecretCache) GenerateSecret(ctx context.Context, connectionID, resourc
 	return ns, nil
 }
 
+func (sc *SecretCache) addFileWatcher(file string, key ConnKey) {
```

Comment:
Please use `connKey` instead of `key` for clarity. `key` can be misinterpreted as the private key.

Reply:
sounds good